src/Sound.cpp

```diff
@@ -16,7 +16,7 @@
 //  https://freesound.org/people/JohnBuhr/sounds/326868/ attack1
 //  https://freesound.org/people/loudernoises/sounds/334169/ attack2
 //  https://freesound.org/people/Ali_6868/sounds/384915/ arrow
-//  https://freesound.org/people/KeshaFilm/sounds/471834/ heal
+//  https://freesound.org/people/GameDevC/sounds/422832/ whetstone
```

Comment:
Could you restore the heal comment? Looks like it was deleted here

Reply:
woops, sure thing!